squad/frontend/templates/squad/metrics.html

```diff
@@ -3,7 +3,7 @@
 
 {% block content %}
 
-  {% include "squad/project-nav.html" %}
+  {% include "squad/build-nav.html" %}
```

Comment:
same as above: metrics (or at least the current "metrics" page) should not be "under" a specific build.